src/applications/vaos/appointment-list/components/ConfirmedAppointmentDetailsPage/index.jsx

```diff
@@ -236,99 +237,96 @@ export default function ConfirmedAppointmentDetailsPage() {
         </>
       )}
 
-      {!!facility &&
-        !isVideo && (
-          <>
-            <h2
-              className="vads-u-font-size--base vads-u-font-family--sans vads-u-margin-bottom--0"
-              data-cy="va-appointment-details-header"
-            >
-              {header}
-            </h2>
-            <VAFacilityLocation
-              facility={facility}
-              facilityName={facility?.name}
-              facilityId={facilityId}
-              isHomepageRefresh
-              clinicFriendlyName={appointment.location?.clinicName}
-              showCovidPhone={showCovidPhone}
-            />
+      {!isVideo && (
```

Comment:
This block only has two changes:

1. I removed the `!!facility` check.
2. I updated the address being passed to the AddToCalendar component